features/referent_space.feature

```diff
@@ -53,8 +53,8 @@ Feature:
     And I should see "Oui"
 
     When I click the "application-detail-0" element
-    Then I should see "Retour à la liste des candidats colistiers"
-    And I should see "Profession : Scientist"
+    And I wait 10 seconds until I see "Retour à la liste des candidats colistiers"
```

Comment:
isn't 10 seconds a bit much ?

Reply:
it's a timeout.

Reply:
:man_facepalming:  you're right my bad !